ResizeZVol.ps1

```diff
@@ -99,12 +99,31 @@ if($no_zvols_created -ne $numberf_of_zvols){
 }
 
 #Function to Resize the zvol
-function Resize-ZVol
+Function Resize-Zvol($size,$numberf_of_zvols)

```

Comment:
@raeesulasad Please check if you can get the zvol name from the Create-ZVol function